src/components/TodoListItem/TodoListItem.tsx

```diff
@@ -0,0 +1,26 @@
+import { FC } from 'react';
+import { TodoItem } from '../TodoItem/TodoItem';
+import { useTodoContext } from '../../utils/hooks/useTodoContext';
+import { TransitionGroup, CSSTransition } from 'react-transition-group';
+import { TempTodo } from '../TodoTemp/TodoTemp';
+
+export const TodoListItem: FC = () => {
```

Comment:
```suggestion
export const TodoList: FC = () => {
```
ListItem sounds like one todo